src/Simulation/Native/src/external/cintrin.hpp

```diff
@@ -35,7 +35,10 @@ inline void permute_qubits_and_matrix(I *delta_list, unsigned n, M & matrix){
 }
 
 inline std::complex<double> fma(std::complex<double> const& c1, std::complex<double> const& c2, std::complex<double> const& a){
-	return c1*c2 + a;
+	// Expanded complex FMA to hard coded access (much faster)
+	double r = (c1.real() * c2.real() - c1.imag() * c2.imag()) + a.real();
```

Comment:
@dbwz8 The use of `._Val[0]` to access the real part doesn't seem to compile on Linux or Mac in our pipeline, so I had to switch back to the `real()` accessor. Does that negate the speedup you were looking for here or is the equation still a benefit?

Reply:
Actually, I can see where this same fix is in the other branch, so it should be safe.